sign_oca/views/menu.xml

```diff
@@ -13,10 +13,11 @@
         sequence="16"
         groups="sign_oca.sign_oca_group_user"
     />
-    <record model="ir.ui.menu" id="sign_oca_settings_menu">
-        <field name="name">Settings</field>
-        <field name="sequence" eval="90" />
-        <field name="parent_id" ref="sign_oca_root_menu" />
-        <field name="groups_id" eval="[(6,0,[ref('sign_oca.sign_oca_group_admin')])]" />
-    </record>
+    <menuitem
```

Comment:
I don't get why changing this, and more in this commit that is unrelated.

Reply:
I honestly don't understand these comments, it's something that has already been changed in v15 in the corresponding PR, but no problem, it's already changed.